internal/setup/setup.go

```diff
@@ -142,6 +163,7 @@ func (r *Release) validate() error {
 	keys := []SliceKey(nil)
 	paths := make(map[string]*Slice)
 	globs := make(map[string]*Slice)
+	generate := make(map[GenerateKind]*Slice)
```

Comment:
I think generate can be a `string` pointing to the path because we dont even want two entries in the same package and another variable to hold the slice for error reporting.

Reply:
This has since been removed. Later used in `setup.Select`. Please let me know if you still have the same comments.